packages/workflow/src/records/state-transitions.ts

```diff
@@ -216,13 +216,8 @@ export async function toCorrectionApproved(
     paymentReconciliation
   )
 
-  const correctionRequestTaskWithPractitionerExtensions = setupLastRegUser(
-    correctedTask,
-    practitioner
-  )
-
-  const correctionRequestWithLocationExtensions = await setupLastRegLocation(
-    correctionRequestTaskWithPractitionerExtensions,
+  const correctedWithLocationExtensions = await setupLastRegLocation(
+    setupLastRegUser(correctedTask, practitioner),
```

Comment:
We should probably use the `withPractitionerDetails` here as it also returns the resources that needs to be added to the bundle.